detection_rules/rule.py

```diff
@@ -971,7 +975,7 @@ def latest_version(self) -> Optional[int]:
     @property
     def autobumped_version(self) -> Optional[int]:
         """Retrieve the current version of the rule, accounting for automatic increments."""
-        version = self.latest_version
+        version = self.latest_version or self.data.version
```

Comment:
I would give precedence to _explicitly_ set

```suggestion
        version = self.data.version or self.latest_version
```